src/cli/task-generate.ts

```diff
@@ -216,7 +216,10 @@ export const getBoilerplateByExtension = (tagName: string, extension: GenerableE
 };
 
 /**
- * Get the boilerplate for a component.
+ * Get the boilerplate for a file containing the definition of a component
+ * @param tagName the name of the tag to give the component
+ * @param hasStyle designates if the component has an external stylesheet or not
+ * @returns the contents of a file that defines a component
```

Comment:
The new rules doesn't require a summary or `@returns` line - I tried to add them as well since I'd like those to be enforced someday